server/actions/Contentful.ts

```diff
@@ -125,7 +125,7 @@ export async function getJournalEntriesByReviewStatus(reviewed: boolean){
         for (var i=0; i < entries.total; i++) {
             var journalEntry: JournalEntry = {};
             journalEntry.id = entries.items[i].sys.id; // unique id for the asset
-            journalEntry.body = entries.items[i].fields.body['en-US'];
+            // journalEntry.body = entries.items[i].fields.body['en-US'];  // we probably dont need the body when getting them all (i may be wrong) -z
```

Comment:
Hmmm, this is a good point since the only place we'd use this function is to show the previews of journals. If all the preview display uses is the title, description, _id, and the image, then some parts can be omitted.

Reply:
I think Kemal's right. If those are the only things needed in the preview, and this is the only thing we use the function for, then we can omit the rest.